### ISSUE TITLE
Recovering after reboot

### ISSUE THREAD

**Issue by ygoodmn** [CiscoCloud/mantl#1572]
ansible 1.9.6
Python 2.7.5
git status# HEAD detached at 1.1.0
Cloud Environament: Openstack onprem (Juno, via Fuel Mirantis 6.1.1)
Terraform v0.6.14

Note: Only have a few hours to trouble shoot this.

Someone powered down all my compute nodes (or a power failure).
Happened Thursday night: 
```
Jun 16 16:06:57 mantlhgl4-control-02 systemd-logind: Power key pressed.
Jun 16 16:06:57 mantlhgl4-control-02 systemd-logind: Powering Off...
```
Came in Sunday morning
Couldn't connect to Mantl UI  and on login to controller /tmp/run was full errors in messages

[messages.txt](https://github.com/CiscoCloud/mantl/files/322150/messages.txt)
[messages-20160617.txt](https://github.com/CiscoCloud/mantl/files/322151/messages-20160617.txt)

Ran 2x ansible-playbook  -e @security.yml playbooks/reboot-hosts.yml
to try to recover  did not recover (i.e. docker ps -a on a controller shows no docker running)

Will add more during the day, as was told could re-run the mantl.yml to see if that recovers it without terraform destroy.
 - Looks like Mesos Slaves are okay, it seems consul not responding.
 - Running ansible-playbook -e @security.yml mantl.yml now
Issue now: So this seems an issue, will check previous issues.
```
failed: [mantlhgl4-control-03] => {"changed": true, "cmd": ["/usr/local/bin/consul-wait-for-leader.sh"], "delta": "0:00:30.122639", "end": "2016-06-19 07:30:38.058046", "rc": 1, "start": "2016-06-19 07:30:07.935407", "warnings": []}
stdout: No Consul leader elected in 30 seconds
```
 - Docker images show nothing, does it need docker images for consul to run
```
$sudo netstat -nap | grep " LISTEN " | grep consul
tcp        0      0 127.0.0.1:8400          0.0.0.0:*               LISTEN      839/consul
tcp        0      0 127.0.0.1:8500          0.0.0.0:*               LISTEN      839/consul
tcp        0      0 127.0.0.1:8600          0.0.0.0:*               LISTEN      839/consul
tcp6       0      0 :::8300                 :::*                    LISTEN      839/consul
tcp6       0      0 :::8301                 :::*                    LISTEN      839/consul
tcp6       0      0 :::8302                 :::*                    LISTEN      839/consul
$sudo docker images
REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
```

Question: Seems there is a try to remove docker images in messages log, is this supposed to happen. (This was before power down)
```
Jun 16 03:39:19 mantlhgl4-control-02 systemd: Started Remove exited docker containers.
Jun 16 03:39:19 mantlhgl4-control-02 systemd: Starting Remove exited docker containers...
Jun 16 03:39:19 mantlhgl4-control-02 systemd: Started Remove unused docker images.
Jun 16 03:39:19 mantlhgl4-control-02 systemd: Starting Remove unused docker images...
Jun 16 03:39:19 mantlhgl4-control-02 bash: Error response from daemon: conflict: unable to delete 683fb468c424 (cannot be forced) - image is being used by running container 6b86282ecaff
Jun 16 03:39:19 mantlhgl4-control-02 bash: Error response from daemon: conflict: unable to delete 1f8eaf095644 (cannot be forced) - image is being used by running container f233890e32c4
Jun 16 03:39:19 mantlhgl4-control-02 bash: Error response from daemon: conflict: unable to delete ed554d01b982 (cannot be forced) - image is being used by running container b6161f4f3871
Jun 16 03:39:19 mantlhgl4-control-02 bash: Error response from daemon: conflict: unable to delete 63d00e82730d (cannot be forced) - image is being used by running container 68d683777e53
```

This of course is an edge case, but this could happen in Production (and has in the past), unless you state you only support Geo-Location of Mantl installation.

**Comment by ygoodmn** [comment]
Issue: Consul /var/lib/consul/raft/peers.json has in it null
Solution:
1. Stop all nodes running consul
 ansible *control-0* -m shell -a 'sudo service consul stop'
2. Modify manaully the peers.json to have all peers.
Example
[
  "10.0.1.8:8300",
  "10.0.1.6:8300",
  "10.0.1.7:8300"
]
3. Restart consul
ansible *control-0* -m shell -a 'sudo service consul start'
---NOW WAITING To see if recovers

**Comment by ygoodmn** [comment]
Next issue was /var/run/fail2ban directory was missing on some nodes, needed to create the directory and restart the component so the ansible-playbook would not return an error.

**Comment by siddharthist** [comment]
Possible duplicate of #566, #1238, #1541

**Comment by siddharthist** [comment]
@ygoodmn Please wrap all logs and code snippets in triple backquotes, like \`\`\`

**Comment by siddharthist** [comment]
@ygoodmn If you see a Consul leader election failure again, please try out https://github.com/CiscoCloud/mantl/pull/948, we're trying to get some more experience testing with it.

**Comment by siddharthist** [comment]
Closed in #948
